packages/typespec-python/src/emitter.ts

```diff
@@ -252,6 +227,12 @@ function getType(context: SdkContext, type: EmitterType): any {
     } else {
         newValue = emitType(context, type);
     }
+
+    if (oriType?.kind === "ModelProperty") {
+        updateWithEncode(context, oriType, newValue);
```

Comment:
with this code, `@clientFormat` will beat out `@encode`, when we want `@encode` to win. We just want to support i someone still has `@clientFormat`, we don't completely break. I'll flip aroudn these two lines of code then i think we're good